tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/taskflow_generator.h

```diff
@@ -43,7 +43,12 @@ class TaskflowGenerator
 public:
   using UPtr = std::unique_ptr<TaskflowGenerator>;
 
+  TaskflowGenerator() = default;
   virtual ~TaskflowGenerator() = default;
+  TaskflowGenerator(const TaskflowGenerator&) = delete;
+  TaskflowGenerator& operator=(const TaskflowGenerator&) = delete;
+  TaskflowGenerator(TaskflowGenerator&&) = delete;
+  TaskflowGenerator& operator=(TaskflowGenerator&&) = delete;
```

Comment:
This one as well. I don't know why you would want to copy it, but it seems like it should be safe to do so.

Reply:
same as above.